example/fedora/repository/40/repositories.yaml

```diff
@@ -1,2 +1,5 @@
-- id: "fedora"
-  metalink: https://mirrors.fedoraproject.org/metalink?repo=fedora-${version}&arch=${architecture}
+otk.define:
++  packages:
```

Comment:
it looks like there are some copy/paste leftover `+` here at the start of the line?